src/features/form/components/AdminFormDetailPages/RepliesPage.module.css

```
@@ -166,6 +173,8 @@
 	padding: 1rem;
 	background-color: var(--selection);
 	border-radius: 1rem;
+	word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)